### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Kigaard**
Fixes typos and removes double spaces.

### CODE REVIEW

File: EDAM_dev.owl [hunk 1]
```diff
@@ -33181,7 +33181,7 @@
         <created_in>1.11</created_in>
         <documentation rdf:resource="http://seqanswers.com/wiki/AFG"/>
         <oboInOwl:hasDbXref rdf:resource="http://seqanswers.com/wiki/AFG"/>
-        <oboInOwl:hasDefinition>AFG is a single text-based file assembly format  that holds read and consensus information together</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>AFG is a single text-based file assembly format hat holds read and consensus information together</oboInOwl:hasDefinition>
```
Comment (matuskalas) [inline EDAM_dev.owl:33184]: One 't' too little 😊

Comment (Kigaard) [reply EDAM_dev.owl:33184]: Oops 😅 Thanks

**Review by matuskalas — CHANGES_REQUESTED** [review]
Awesome, thank you a million for fixing these! 🙏🏽👾

I hope no more double-spaces ae left in EDAM😊

Could you please when doing this clean-up also check if there are any newline characters (\r or \n, _i.e._ CR or LF) inside XML elements or attributes?

And please one last spell-check. By plain eye I found 1 typo (commented in the code).

Million thanks again! 🙌🏽 Your attention to detail is highly appreciated👍🏽

**Comment by matuskalas** [comment]
> Could you please when doing this clean-up also check if there are any newline characters (\r or \n, _i.e._ CR or LF) inside XML elements or attributes?

And Tabs! (\t)

Thank youuu🙏🏽

**Comment by Kigaard** [comment]
@matuskalas, the file has been checked and I have been unable to find any typos except in the `AGF`-term.
By a simple `Find`, there are not any \n, \r, \t that I could find.

File: EDAM_dev.owl (hunk 1 above)
Comment (Kigaard) [reply EDAM_dev.owl:33184]: It should be fixed now 😄 I thought you changed it by the comment 😅

Comment (matuskalas) [reply EDAM_dev.owl:33184]: Awesome, thanks👍🏽

**Review by matuskalas — APPROVED** [review]
🙌🏽

**Pull request merged**